src/test/java/core/basesyntax/service/impl/PurchaseFruitHandlerImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.storage.DataBase;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class PurchaseFruitHandlerImplTest {
+    private static final Map<String, Integer> db = new HashMap<>();
+    private static final AddHandlerImpl addHandler = new AddHandlerImpl();
+    private static final PurchaseFruitHandlerImpl purchaseHandler
+            = new PurchaseFruitHandlerImpl();
+    private static OperationType operationTypeSupply;
+    private static OperationType operationTypeBalance;
```

Comment:
why they're static if you don't use Before ?

Reply:
Fixed